build.gradle

```diff
@@ -44,10 +44,13 @@ plugins {
 apply plugin: 'com.palantir.baseline'
 apply plugin: 'com.palantir.git-version'
 
+version System.env.CIRCLE_TAG ?: gitVersion()
```

Comment:
I had a 25second gradle configuration time: https://scans.gradle.com/s/6j5nbkrgpj54u/performance/configuration#summary-total

Reply:
Can we make these improvements in separate PRs? Unix philosophy :-)

Reply:
Pulled out to https://github.com/palantir/dialogue/pull/365